Client/e2e/protractor.conf.js

```diff
@@ -0,0 +1,28 @@
+// Protractor configuration file, see link for more information
```

Comment:
Was it auto generated? and all the tests too?
If so, I think you should delete it, it's just adding noise